src/clj_http_hystrix/core.clj

```diff
@@ -74,14 +77,6 @@
       (warn exception message)
       (warn message))))
 
-(defn status-codes
```

Comment:
Ah yes, this was intended to be a useful utility that would create a bad-request predicate for any set of status codes.